packages/react-native-web/src/modules/AccessibilityUtil/isDisabled.js

```diff
@@ -7,6 +7,10 @@
  * @flow
  */
 
-const isDisabled = (props: Object) => props.disabled || props['aria-disabled'];
+const isDisabled = (props: Object) =>
+  props.disabled ||
+  (Array.isArray(props.accessibilityStates) &&
+    props.accessibilityStates.indexOf('disabled') > -1) ||
+  props['aria-disabled'];
```

Comment:
Drop this check. Add comment to release notes that any use of `aria-disabled` needs to move to `accessibilityStates`